src/app/public/components/loginpage/loginpage.component.html

```diff
@@ -1,28 +0,0 @@
-<div class="form-container mat-elevation-z5">
```

Comment:
It looks like the merge conflict is coming from the fact that the file is empty? This is weird. I'm looking into it.

Reply:
We need to delete this file from the commit. It's causing issues merging and I am unsure of how to resolve it without manually deleting it, but it's one of Jim's files. Whenever you get a chance can you check this out?

Reply:
I also have no idea where the matching .ts file is for this file, so I'm majorly lost on that issue with the merge conflict.